karma/percy.md

```diff
@@ -0,0 +1,9 @@
+# Running Percy
+
+These are temporary steps until we set up the CI.
```

Comment:
Configuration is getting along well.

#1864 runs Karma on CI and uploads screenshots to Percy.

`PERCY_TOKEN` secret has been added to the repository.

Reply:
Awesome!

Reply:
I'll remove this file right away then.